types/protocol.go

```diff
@@ -0,0 +1,11 @@
+package types
+
+// Constants to match up protocol versions and messages
+const (
+	eth63 = 63
+	eth64 = 64
```

Comment:
in what cases would these 2 be used? looks like eth65 is the only one used rn

Reply:
Good question. Idk, I just copied them over. Do you think we should remove them?

Reply:
if they aren't used we can probably remove them for now